python/baseline/pytorch/seq2seq/decoders.py

```diff
@@ -115,14 +106,14 @@ def __init__(self, tgt_embeddings, **kwargs):
         else:
             self.input_i = self._basic_input
         pdrop = kwargs.get('dropout', 0.5)
-        self.decoder_rnn = pytorch_rnn_cell(dsz, self.hsz, rnntype, layers, pdrop)
+        self.decoder_rnn = rnn_cell(dsz, self.hsz, rnntype, layers, pdrop)
         self.dropout = torch.nn.Dropout(pdrop)
         self.init_attn(**kwargs)
 
         do_weight_tying = bool(kwargs.get('tie_weights', False))
         is_valid_tying = self.hsz == self.tgt_embeddings.get_dsz()
 
-        self.preds = pytorch_linear(self.hsz, self.tgt_embeddings.get_vsz())
+        self.preds = torch.nn.Linear(self.hsz, self.tgt_embeddings.get_vsz())
```

Comment:
Using `torch.nn.Linear` directly loses the initialization we do in `pytorch_linear` like setting the bias to `0`. Do we want to remove that?